cdk/drupal/drupal_stack.py

```diff
@@ -393,9 +405,9 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             "Properties.VpcId",
             {
                 "Fn::If": [
-                    customer_vpc_not_given_condition.logical_id,
+                    vpc_not_given_condition.logical_id,
                     vpc.ref,
-                    customer_vpc_id_param.value.to_string()
+                    vpc_id_param.value.to_string()
```

Comment:
In other places in the code, we are using the parameter.value_as_string attribute.

Reply:
I updated this most places but we still need to use `value.to_string()` for `Number` param types.